src/drivers/mct/cime_config/testdefs/testlist_drv.xml

```diff
@@ -614,12 +614,20 @@
       </machine>
     </machines>
   </test>
+  <test name="DAE" grid="ww3a" compset="ADWAV">
+    <machines>
+      <machine name="hobart" compiler="nag" category="prealpha"/>
+    </machines>
+    <options>
+      <option name="wallclock"> 00:20:00 </option>
+    </options>
+  </test>
   <test name="PRE" grid="f09_f09" compset="ADESP">
     <machines>
       <machine name="hobart" compiler="nag" category="pauseresume"/>
     </machines>
     <options>
-      <option name="wallclock"> 00:20 </option>
+      <option name="wallclock"> 00:20:00 </option>
```

Comment:
Is this right?

Reply:
yes

Reply:
At one point, I thought we were standardizing times so they were platform independent. Is that not the case? I do not think I had an issue with this (the issue I was having turned out to be a different problem).
I'm happy to back this out if it is not an improvement.

Reply:
We are - yellowstone was the exception, all others are hh:mm:ss